### ISSUE TITLE
[Feature] Add Rancher to Container Orchestration

### ISSUE THREAD

**Issue by adamgoose** [kamranahmedse/developer-roadmap#160]
[Rancher](http://rancher.com/) is a new Container Orchestration platform that might be worth adding to the DevOps map.

**Comment by DarkMagic** [comment]
That's right. Rancher 2.x provide unified management platform for all kinds of cluster.
The clusters that it supports: AKS, EKS, GKE, RKE on Bare Metal / On-premises clusters.
https://rancher.com/docs/rancher/v2.x/en/overview/
